Gemfile

```diff
@@ -34,6 +34,7 @@ gem 'sprockets-rails', '>= 2.3.3', '< 3'
 gem 'responders', '~> 2.0'
 gem 'roadie-rails', '~> 1.1'
 gem 'x-editable-rails', '~> 1.5.5'
+gem 'foreman-tasks', '>= 0.7.20'
```

Comment:
I agree, but we need to find a way to extract single deps from the bundle so we can move slowly migrate to properly packaged deps, otherwise it means we can never move from where we are